src/styleguide/components/s-api-mock-test.vue

```diff
@@ -273,26 +273,25 @@
               // Do nothing
             }
 
-            const resolver: ResponseResolver = (req, res, ctx: any) => res( // eslint-disable-line @typescript-eslint/no-explicit-any
-              ctx.status(parseInt(configuration.status, 10)),
-              response ? ctx.json(response) : null
-            );
+            const resolver: ResponseResolver = () => HttpResponse.json(response as object, {
```

Comment:
casting as an object probably can be prevented, but its also not to critical to cast here. So fill free to take the time for improving it or leave it as it.